### PR TITLE
Dev/dr/reactive fixes

### PR DESCRIPTION
**Pull request by dr1rrb**
## Bugfix and Feature

- [Generator] Prevent member name conflicts in gen'ed code
- Auto resolve dispatcher in VM and Command to allow cre…
- Use DispatcherQueue instead of Dispatcher in SmoothVSM

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/Uno.Extensions.Reactive.UI/Utils/Events/EventManager.THandler.cs [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+		};
+
+	/// <inheritdoc />
+	public void Dispose()
+	{
+		// TODO: Dispose All
+	}
+}
+
+//internal interface IInvocationList<in THandler, in TArgs> : IEventManager<THandler, TArgs>
```
Comment (davidjohnoliver) [inline src/Uno.Extensions.Reactive.UI/Utils/Events/EventManager.THandler.cs:84]: Commented code below, can it be removed?

File: src/Uno.Extensions.Reactive.UI/Utils/Events/EventManager.THandler.cs [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+		{
+			(not null, true) => new CoalescingDispatcherInvocationList<THandler, TArgs>(_owner, _raiseMethod, dispatcher),
+			(not null, false) => new QueueingDispatcherInvocationList<THandler, TArgs>(_owner, _raiseMethod, dispatcher),
+			_ => new MultiThreadInvocationList<THandler, TArgs>(_owner, _raiseMethod)
+		};
+
+	/// <inheritdoc />
+	public void Dispose()
+	{
+		// TODO: Dispose All
```
Comment (davidjohnoliver) [inline src/Uno.Extensions.Reactive.UI/Utils/Events/EventManager.THandler.cs:80]: should anything be disposed here?

**Review by davidjohnoliver — APPROVED** [review]

File: src/Uno.Extensions.Reactive.UI/Presentation/Bindings/BindableViewModelBase.cs [hunk 3]
```diff
@@ -54,37 +63,52 @@ protected BindablePropertyInfo<TProperty> Property<TProperty>(string propertyNam
...
-				 }
-				 catch (Exception error)
-				 {
-					 this.Log().Error(
-						 $"Synchronization from ViewModel to View of '{propertyName}' failed."
-						 + "(This is a final error, changes made in the VM are no longer propagated to the View.)",
-						 error);
-				 }
-			 });
+		async void ViewModelToView(Action<TProperty?> updated)
```
Comment (jeromelaban) [inline src/Uno.Extensions.Reactive.UI/Presentation/Bindings/BindableViewModelBase.cs:66]: Should it be cancellable?

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.Extensions.Reactive.UI/Presentation/Bindings/BindableViewModelBase.cs (hunk 3 above)
Comment (dr1rrb) [reply src/Uno.Extensions.Reactive.UI/Presentation/Bindings/BindableViewModelBase.cs:66]: On dispose the `stateImpl` will be disposed, which will end the `source` async enumerable. We could however also cancel the async dispatcher resolution ... pushing a new commit for that!

**Pull request merged**